docs/src/views/IntegrateCustomComponents.tsx

```diff
@@ -229,6 +237,45 @@ export default () => (
       For more advanced theming scenarios, please take a look in the <b>Variables</b> section on the{' '}
       <NavLink to="theming">Theming guide</NavLink>.
     </p>
+
+    <Header
+      as="h3"
+      content={
+        <>
+          Example 3. Using <code>accessibility</code> property
+        </>
+      }
+    />
+    <p>
+      It is possible to define accessibility behavior of the created component - in order to do that
+      the <code>accessibility</code> prop should be defined as part of <code>defaultProps</code>.
+    </p>
+    <p>
+      In the following example <code>buttonBehavior</code> is provided to <code>StyledButton</code>{' '}
+      that results in accessibility attributes being evaluated for the component - and those,
+      subsequently, are consumed in the <code>render</code> function:
+    </p>
+    <ExampleSnippet
+      value={`
+        import { buttonBehavior } from '@stardust-ui/react'
+
+        const StyledButton = createComponent({
+          displayName: 'StyledButton',
+          defaultProps: {
+            accessibility: buttonBehavior
+          },
+          render: ({stardust, className, children}) => {
+            const { classes, accessibility } = stardust
+            return <button {...accessibility.root} className={classes.root}>{children}</button>
```

Comment:
should this be `{...accessibility.attributes.root}`?

Reply:
thank you!